src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -70,5 +70,13 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
             }
             return false;
         }
+
+        public static bool ShouldDisplayReadOnly(this IPropertySymbol property)
```

Comment:
>ShouldDisplayReadOnly [](start = 27, length = 21)

If this extension is only used by `SymbolDisplay`, consider moving there as a `private` method.

Reply:
This will end up being used for MetadataAsSourceService as well in the near future. At that time I'll find an appropriate permanent home for it.

Reply:
Since this comment was made, MetadataAsSourceService wound up diverging from SymbolDisplay--specifically, we're "reasonable imitation of the original signature" is a goal for MetadataAsSourceService, but the goal for SymbolDisplay is really "show the informative bits of the signature for the context, then stop".

Reply:
I will take another look at moving it given that.

Reply:
Done